openshift_pool/openshift/stack.py

```diff
@@ -75,6 +81,8 @@ def _config_domains(self, stack, method, check_connection_attempts=10):
                     ):
                 return
             connection_attempts += 1
+            time.sleep(20)
```

Comment:
Add comment on why we are sleeping here for 20 seconds.

Reply:
Done